app/controllers/concerns/form_errors.rb

```diff
@@ -0,0 +1,32 @@
+module FormErrors
```

Comment:
can you add a brief documentation comment here explaining how this module connects to the rest of the app?